common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -19,6 +19,22 @@ h2 {
   }
 }
 
+div.feedback_hint_correct {
```

Comment:
Unfortunately, our platform variables are not available in this file, so please add these at the top of the file (after $annotation-yellow):
 
```
$color-copy-tip: rgb(100,100,100);
$color-success: rgb(0, 136, 1);
$color-fail: rgb(212, 64, 64);
```

and then update your colors in the classes like so:

```
div.feedback_hint_correct {
    color: $color-success;
}

div.feedback_hint_incorrect {
    color: $color-fail;
}

div.feedback_hint_text {
    color: $color-copy-tip;
}

div.problem_hint {
    color: $color-copy-tip;
    margin-bottom: 20px;
}
```

That was we can at least have a shot at keeping these colors in sync throughout the platform.